platforms/ios/src/TangramMap/platform_ios.mm

```diff
@@ -94,6 +95,15 @@ void initGLExtensions() {
     return data;
 }
 
+std::string iOSPlatform::getAssetPath(const Tangram::Url& _path) const {
+    if (_path.isAbsolute()) {
```

Comment:
What covers this use case? iOS client application read scope is only within the bundle.